langchain4j-mcp/src/test/java/dev/langchain4j/mcp/registryclient/McpRegistryClientTest.java

```diff
@@ -0,0 +1,108 @@
+package dev.langchain4j.mcp.registryclient;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import dev.langchain4j.mcp.registryclient.model.McpRegistryHealth;
+import dev.langchain4j.mcp.registryclient.model.McpRegistryPong;
+import dev.langchain4j.mcp.registryclient.model.McpServer;
+import dev.langchain4j.mcp.registryclient.model.McpServerList;
+import dev.langchain4j.mcp.registryclient.model.McpServerListRequest;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * TODO: this test runs against the official MCP registry at https://registry.modelcontextprotocol.io/
+ * We might want to stop depending on it and instead run our own subregistry when we have a suitable SDK
+ * available to do that for testing. Another option is to introduce a mock that substitutes it.
+ */
+public class McpRegistryClientTest {
+
+    private static final Logger log = LoggerFactory.getLogger(McpRegistryClientTest.class);
+
+    static DefaultMcpRegistryClient client;
+
+    @BeforeAll
+    public static void prepare() {
+        client = new DefaultMcpRegistryClient.Builder()
```

Comment:
```suggestion
        client = DefaultMcpRegistryClient.builder()
```
nit: would be nice to have this convenience method